proto/substrait/algebra.proto

```diff
@@ -410,6 +412,13 @@ message DdlRel {
   // The body of the CREATE VIEW
   Rel view_definition = 7;
 
+  // Field that could be used to extend or introduce any target specific metadata about the table,
+  // like replication, partition, etc.
+  ExtensionObject table_metadata = 8;
+
+  // Indices of all primary keys in the table
+  repeated int32 primary_keys = 9;
```

Comment:
What source do the int32 values refer to?

Reply:
Also, update the comment below saying to add primary keys to the message.

Reply:
Assuming that the table columns are listed in table_schema 
(table_schema.struct.types)
as a list of columns types 
so the int32 value refer to the index of the column (type) in the list